example-es6/src/modules/demo.js

```diff
@@ -0,0 +1,53 @@
+import remoteDOM from 'remoteDOM'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './app'
+
+self.window = remoteDOM.remote.window
+self.document = remoteDOM.remote.document
+remoteDOM.remote.setChannel(self)
+
+class AppContainer {
```

Comment:
The naming here is a bit confusing. AppContainer sounds like it's an actual component (also because of the Title Case). Isn't it just a propsBuilder method?